Makefile

```
@@ -1,45 +1,45 @@
-current_user  := $(shell id -u)
-current_group := $(shell id -g)
-BUILD_DIR     := $(PWD)
-DOCKER_FLAGS  := --interactive --tty
+current_user         := $(shell id -u)
+current_group        := $(shell id -g)
+BUILD_DIR            := $(PWD)
+DOCKER_FLAGS         := --interactive --tty
 UNIQUE_APP_CONTAINER := $(shell uuidgen)-app
+COVERAGE_FLAGS       := --coverage-html coverage
+DEFAULT_GOAL         := ci
 
-.PHONY: ci test phpunit cs stan composer
+install-php:
+	docker run --rm $(DOCKER_FLAGS) --volume $(BUILD_DIR):/app -w /app --volume ~/.composer:/composer --user $(current_user):$(current_group) composer install --ignore-platform-reqs $(COMPOSER_FLAGS)
+
+update-php:
+	docker run --rm $(DOCKER_FLAGS) --volume $(BUILD_DIR):/app -w /app --volume ~/.composer:/composer --user $(current_user):$(current_group) composer update --ignore-platform-reqs $(COMPOSER_FLAGS)
 
-DEFAULT_GOAL := ci
+clear:
+	rm -rf var/cache/
+	docker-compose run --rm --no-deps app rm -rf var/cache/
+
+# n alias to avoid frequent typo
+clean: clear
+
+phpunit:
+	docker-compose run --rm app ./vendor/bin/phpunit
+
+phpunit-with-coverage:
+	docker-compose -f docker-compose.yml -f docker-compose.debug.yml run --rm --name $(UNIQUE_APP_CONTAINER)-$@ app_debug ./vendor/bin/phpunit --configuration=phpunit.xml.dist $(COVERAGE_FLAGS)
```

```suggestion
	docker-compose -f docker-compose.yml -f docker-compose.debug.yml run --rm app_debug ./vendor/bin/phpunit --configuration=phpunit.xml.dist $(COVERAGE_FLAGS)
```
You can remove `UNIQUE_APP_CONTAINER` both here and from the initial variables. It is a vestige copied from a old version of the Makefile of the Fundraising App, where it allowed to run `make` with the `-j` parameter to run all CI tasks in parallel. This was supposed to speed things up, but in reality it slowed things down (because we did not have multiple processors in CI) and added too much complication to the docker commands in the Makefile.